utils/utils.js

```diff
@@ -630,6 +630,14 @@ const getDataFromSource = (src, dest, properties) => {
   return data;
 };
 
+const getConfigUrlWithWritekey = (writeKey) => {
+  if (writeKey) {
+    return `https://api.rudderlabs.com/sourceConfig/?p=process.module_type${writeKey}&v=process.package_version`;
+  } else {
+    return CONFIG_URL;
+  }
```

Comment:
We need to send the write key as a new query parameter
```suggestion
  // Add write key as a new query parameter if it is valid
  return CONFIG_URL.concat(CONFIG_URL.includes("?") ? "&" : "?").concat(writeKey ? `writeKey=${writeKey}` : "");
```

Reply:
done